src/platform/sdl/CMakeLists.txt

```diff
@@ -15,7 +15,8 @@ endif()
 if(SDL_VERSION EQUAL "1.2" OR NOT SDL2_FOUND)
 	find_package(SDL 1.2)
 	if(SDL_FOUND)
-		set(SDL_VERSION "1.2" PARENT_SCOPE)
+		set(SDL_VERSION "1.2")
+		set(SDL_VERSION "${SDL_VERSION}" PARENT_SCOPE)
```

Comment:
Should still be moved to the root block below the return (unless that breaks things).